init/miniaf.rc

```diff
@@ -0,0 +1,4 @@
+service miniaf /usr/libexec/droid-hybris/system/bin/miniafservice
```

Comment:
Should this be somewhere else? It's not built here.

Reply:
Hm yes I think so. I'll move it.